components/dropdown/test/dropdown-content.html

```diff
@@ -0,0 +1,303 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+	<title>d2l-dropdown unit tests</title>
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="/node_modules/mocha/mocha.js"></script>
+	<script src="/node_modules/chai/chai.js"></script>
+	<script src="/node_modules/@polymer/test-fixture/test-fixture.js"></script>
+	<script src="/node_modules/wct-mocha/wct-mocha.js"></script>
+	<script type="module" src="../dropdown.js"></script>
+	<script type="module" src="../dropdown-content.js"></script>
+</head>
+
+<body>
+
+	<test-fixture id="dropdown">
+		<template>
+			<div>
+				<div id="optionallyFocusable">
+					<d2l-dropdown>
+						<button class="d2l-dropdown-opener"></button>
+						<d2l-dropdown-content>
+							<p id="non_focusable_inside">a</p>
+							<a id="focusable_inside" href="http://www.desire2learn.com">b</a>
+						</d2l-dropdown-content>
+					</d2l-dropdown>
+					<p id="non_focusable_outside">c</p>
+					<button id="focusable_outside">out here</button>
+				</div>
+			</div>
+		</template>
+	</test-fixture>
+
+	<script type="module">
+		describe('d2l-dropdown', () => {
+
+			let dropdown, opener, content;
+
+			beforeEach(() => {
+				dropdown = fixture('dropdown');
+				opener = dropdown.querySelector('.d2l-dropdown-opener');
+				content = dropdown.querySelector('d2l-dropdown-content');
+			});
+
+			describe('events', () => {
+
+				it('fires open event when open attribute is added', done => {
+					content.addEventListener('d2l-dropdown-open', () => {
+						expect(content.opened).to.be.true;
+						done();
+					});
+					content.setAttribute('opened', true);
+				});
+
+				it('fires close event when open attribute is removed', done => {
+					content.addEventListener('d2l-dropdown-open', () => {
+						content.removeAttribute('opened');
+					});
+					content.addEventListener('d2l-dropdown-close', () => {
+						expect(content.opened).to.be.false;
+						done();
+					});
+					content.setAttribute('opened', true);
+				});
+
+			});
+

```

Comment:
The [width](https://github.com/BrightspaceUI/dropdown/blob/master/test/dropdown-content.html) tests were removed and converted to visual diff tests.